server/controllers/userController.js

```diff
@@ -116,15 +116,15 @@ class UserController {
 
   static async verifyEmailLink(req, res) {
     try {
-      const { token } = req.query;
-      const payload = auth.verifyMailToken(token);
-
-      if (!payload) return util.errorStatus(res, 400, 'Invalid Verification Link');
-
-      const { email } = payload;
-
+      const { token, id } = req.query;
+      const user = await models.Users.findByPk(id);
```

Comment:
What is "findByPk"?

Reply:
Sequelize is deprecating findById in v5 I think and are replacing it with findByPK(). PK = Primary Key.

Reply:
Oh, I see. I wasn't aware of that.